packages/synapse-react-client/src/components/SynapseTable/TopLevelControls/TopLevelControls.tsx

```diff
@@ -64,13 +72,25 @@ const TopLevelControls = (props: TopLevelControlsProps) => {
     cavaticaHelpURL,
   } = props
 
+  const rowSelectionCustomControls = getFilteredCustomControls(
+    true,
+    customControls,
+  )
+  const topLevelCustomControls = getFilteredCustomControls(
+    false,
+    customControls,
+  )
```

Comment:
I probably would have used lodash [partition](https://lodash.com/docs/4.17.15#partition), but having a custom utility like you have here is likely more readable.

Reply:
Changed to use partition (less code, and less code to test)